### ISSUE TITLE
[BUG] Standardize advantage twice in on-policy buffer and vector-on-policy buffer

### ISSUE THREAD

**Issue by r-y1** [PKU-MARL/omnisafe#170]
### Required prerequisites

- [X] I have read the documentation <https://omnisafe.readthedocs.io>.
- [X] I have searched the [Issue Tracker](https://github.com/PKU-MARL/omnisafe/issues) and [Discussions](https://github.com/PKU-MARL/omnisafe/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/PKU-MARL/omnisafe/discussions/new).

### What version of OmniSafe are you using?

0.1.1

### System information

No need

### Problem description

I found that the advantage will be standardized twice in the on-policy buffer and vector-on-policy buffer. I believe this is a bug.
See below:
```python
    def get(self) -> Dict[str, torch.Tensor]:
        """Get the data in the buffer."""
        self.ptr, self.path_start_idx = 0, 0

        data = {
            'obs': self.data['obs'],
            'act': self.data['act'],
            'target_value_r': self.data['target_value_r'],
            'adv_r': self.data['adv_r'],
            'logp': self.data['logp'],
            'discounted_ret': self.data['discounted_ret'],
            'adv_c': self.data['adv_c'],
            'target_value_c': self.data['target_value_c'],
        }

        # self.data['adv_r'] = torch.zeros_like(self.data['adv_r'])
        # self.data['adv_c'] = torch.zeros_like(self.data['adv_c'])

        adv_mean, adv_std, *_ = distributed.dist_statistics_scalar(data['adv_r'])
        cadv_mean, *_ = distributed.dist_statistics_scalar(data['adv_c'])
        if self._standardized_adv_r:
            data['adv_r'] = (data['adv_r'] - adv_mean) / (adv_std + 1e-8)
        if self._standardized_adv_c:
            data['adv_c'] = data['adv_c'] - cadv_mean

        return data
```
```python
    def get(self) -> Dict[str, torch.Tensor]:
        """Get the data from the buffer."""
        data_pre = {k: [v] for k, v in self.buffers[0].get().items()}
        for buffer in self.buffers[1:]:
            for k, v in buffer.get().items():
                data_pre[k].append(v)
        data = {k: torch.cat(v, dim=0) for k, v in data_pre.items()}

        adv_mean, adv_std, *_ = distributed.dist_statistics_scalar(data['adv_r'])
        cadv_mean, *_ = distributed.dist_statistics_scalar(data['adv_c'])
        if self._standardized_adv_r:
            data['adv_r'] = (data['adv_r'] - adv_mean) / (adv_std + 1e-8)
        if self._standardized_adv_c:
            data['adv_c'] = data['adv_c'] - cadv_mean

        return data
```

### Reproducible example code

The Python snippets:

```python

```

Command lines:

```bash

```

Extra dependencies:

```text

```

Steps to reproduce:

1.
2.
3.

**Comment by Gaiejj** [comment]
Thanks for your question!

![image](https://user-images.githubusercontent.com/108712610/226798705-86c28276-97d6-4d26-8bc9-1f5bedf402a9.png)

The current vector buffer is somewhat misleading. In fact, when creating ``OnPolicyBufffer``, we specified ``standardized_adv_r=False`` and ``standardized_adv_c=False``. So only one normalization operation will be performed in the vector buffer. Thanks for your question, we will improve the readability of the code in subsequent versions to avoid similar misunderstandings. You are welcome to keep in touch with us during the subsequent usage.
